src/test/regress/Makefile

```diff
@@ -10,6 +10,10 @@ ifndef MAJORVERSION
 MAJORVERSION := $(basename $(VERSION))
 endif
 
+ifeq ($(TESTHLL),true)
+HLLEXT := "--load-extension=hll"
```

Comment:
undefined_file is probably a better exception to catch, that'll not trigger for extension script failures and such.